types/textfit/textfit-tests.ts

```diff
@@ -0,0 +1,16 @@
+import textFit from "textfit";
```

Comment:
I had read [TypeScript: Documentation - Library Structures](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html).

Should I use require keyword or globally import for UMD module?

```suggestion
import textfit = require("textfit")
```

```suggestion
/// <reference types="someLib" />
require("textfit")
```

Reply:
I would write tests with `require`.